libs/ui-lib-tests/cypress/integration/use-cases/create-cluster/with-external-partner-integrations.cy.ts

```diff
@@ -42,6 +43,12 @@ describe('Create a new cluster with external partner integrations', () => {
       ClusterDetailsForm.customManifestsControl.findCheckbox().should('be.checked');
     });
 
+    it('External partner integrations checkbox is unselected after OCP < v4.14 is selected', () => {
```

Comment:
The test would have passed before fixing the bug.
It should first select 4.14, select OCI, then change the version and see that the checkbox is unselected.

We'll need to include one such a test in the new MR with the dropdown.